### DISCUSSION TITLE
How to add theme to WDD Themes Website?

### DISCUSSION THREAD

**Discussion by alchem1no** [t1m0thyj/WinDynamicDesktop#595] [Q&A] [answered]
Hello everyone 👋
I made a [theme](https://www.mediafire.com/file/33o4148u34kxtl1/Fern_Leaves.ddw/file) for this program and would like to see it on the WDD Site. Is that possible to do?

**Comment by t1m0thyj** [comment, marked as answer]
@Myron472 Yes, would be happy to add it to the site 🙂 Please fill out the form following the steps here: https://github.com/t1m0thyj/WDD-website?tab=readme-ov-file#submit-your-theme-to-the-website
